### PR TITLE
UXODS-94 Call To Action (Link) [Code

### PR DESCRIPTION
**Pull request by rs16princeton**
* The colors can be changed based on the figma design. I haven't been able to locate the styles for the links so for now just put some colors in the colors template.

* I will be working on the icons next for the icons JIRA so will work on changing the external icon when I work on that.

### CODE REVIEW

File: src/scss/_color.scss [hunk 1]
```diff
@@ -56,6 +56,12 @@ $emc-color-map: (
         emc-neon: (background-color: #E1FF32, color: #000),
         emc-lime: (background-color: #C9FB36, color: #000),
         emc-olive: (background-color: #A0BD00, color: #000),
+
+        emc-link: (background-color: #fff, color: #0645AD),
+        emc-link-hover: (background-color: #fff, color: #005760),
+        emc-link-active: (background-color: #fff, color: #005760),
+        emc-link-focus: (background-color: #fff, color: #005760),
+        emc-link-visited: (background-color: #fff, color: #0B0080),
```
Comment (mzarillopu) [inline src/scss/_color.scss]: The _colors.scss file should only reference colors.  Having "*-link-*" in the name seems to indicate that these reference do not belong here.  The colors in this file should already include all of the colors provided in the Figma document.  I would recommend removing these definitions and changing the references in the theme to point to whichever color is closest to the ones that you had defined here.

File: src/scss/_theme_colors_default.scss [hunk 2]
```diff
@@ -12,6 +12,12 @@ $emc-theme-feature-colors: (
 
         body-text: emc-black,
 
+        link: emc-link,
+        link-visited: emc-link-visited,
+        link-hover: emc-link-hover,
+        link-active: emc-link-active,
+        link-focus: emc-link-focus,
```
Comment (mzarillopu) [inline src/scss/_theme_colors_default.scss]: Change these references to point to one of the existing colors in the palette instead of defining new colors.

**Review by mzarillopu — CHANGES_REQUESTED** [review]
See my comments about the color references.  The "colors" file should just reference colors, not features.  Try to match existing colors to the ones you had defined.  If we need other colors defined, I am sure that Jes can come up with some new ones to meet our needs.

File: src/scss/_color.scss (hunk 1 above)
Comment (rs16princeton) [reply src/scss/_color.scss]: I see what you mean. I will just use the blue ones for now then until Jes defines the color palette for these.

**Pull request merged**